Game/level.gd

```diff
@@ -115,3 +115,68 @@ func _on_multiplayer_timer_timeout():
 	if player != null:
 		var sprite = player.get_node("AnimatedSprite2D") as AnimatedSprite2D
 		$MultiplayerController.send_player_info(Preferences.get_player_name(), sprite.animation, sprite.frame, player.position, sprite.flip_h)
+
+func _on_quit_window_close_requested():
+	get_tree().paused = false
+	$QuitMenu/QuitWindow.hide()
+
+func _on_volumeslider_value_changed(value: float):
+	$AudioStreamPlayer.set_volume_db(value - 100)
+
+func _on_color_check_toggled(toggled: bool):
+	if toggled:
+		ProjectSettings.set_setting("rendering/environment/defaults/color_blind_mode", true)
+		var level = get_node("GeneratedLevel")
+		for child in level.get_children():
+			if child.get_child(1).is_in_group("Red"):
+				child.get_child(0).material = ShaderMaterial.new()
```

Comment:
still not a fan of accessing children by index, would rather have them accessed by name (you can set the names of the nodes when they are created in the level generator)

Reply:
Mission complete o7.